static/src/javascripts/projects/common/views/contributions-embed.html

```diff
@@ -0,0 +1,3 @@
+<figure class="element element-interactive contribute-embed interactive element--inline" data-interactive="https://interactive.guim.co.uk/embed/iframe-wrapper/0.1/boot.js" data-canonical-url="https://interactive.guim.co.uk/contributions-embeds/embed/embed.html" data-alt="Contribute">
```

Comment:
it's only an AB test so less important if it's not going out for real, but fyi @gtrufitt is doing a lot of work to get rid of jumping (yank) in articles, so if possible specify a width and a height in the view that's rendered server side it should be done when it's non-ABified (don't let that stop it going out now though)